custom/icds_reports/static/js/topojsons/district_topojson_data.json

```diff
@@ -643,14 +642,20 @@
       "Kulgam",
       "Bandipore",
       "Udhampur",
-      "Leh(Ladakh)",
       "Anantnag",
       "NA",
       "Badgam",
       "Kupwara"
     ],
     "file_name": "jk_blocks_v3.topojson"
   },
+  "Ladakh": {
+    "districts": [
+      "Kargil",
+      "Leh(Ladakh)"
+    ],
+    "file_name": "jk_blocks_v3.topojson"
```

Comment:
I didn't bother making a new file for ladakh yet because it would have been annoying and it doesn't really matter so long as the file still contains the relevant districts.